api/announcements.go

```diff
@@ -17,6 +18,23 @@ var (
 	authToken = config.AuthToken()
 )
 
+func CheckAnn() {
+	jsonFile, err := os.Open("./data/options.json")
+	if err != nil {
+		fmt.Println(err)
+	}
+	defer jsonFile.Close()
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+	var result map[string]interface{}
+	json.Unmarshal([]byte(byteValue), &result)
+	res := result["Announcements"]
+	if res == "true" {
+		Announcements()
+	} else {
+		log.Println("Announcements disabled")
+	}
+}
+
```

Comment:
I told you to just get the config from viper. I know you already did some initial work on using viper instead of json, but just saying.

Reply:
Honestly i'm just reusing your code, nothing much

Reply:
Oh right, i get it now, hold on i'll commit in a few minutes.